charts/argo-cd/README.md.gotmpl

```diff
@@ -104,6 +104,12 @@ For full list of changes please check ArtifactHub [changelog].
 
 Highlighted versions provide information about additional steps that should be performed by user when upgrading to newer version.
 
+### 5.35.0
+This version supports Kubernetes version from `>=1.22.0-0` to `>=1.23.0-0`. Though the supported version of Kubernetes is v1.24 or later, we align with Amazon EKS because there are a lot of EKS users.
```

Comment:
Is this a typo and should it be `>=1.23.0-0` instead of both. Reads a bit confusing now :thinking:

Reply:
Apologies for missing this on the review. My brain filled in the words instead of reading them :)

Reply:
I fixed at https://github.com/argoproj/argo-helm/pull/2087/commits/2b2e351ae1fa33d5f2ea8e0be7a6675ad9fe3a1f 🙋